### PR TITLE
chore: create new locales folder for indonesian

### PR DESCRIPTION
**Pull request by RafaelDavisH**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
Tested locally and this works, but blocked due to the missing Indonesian `translation.json` file.

@naomi-lgbt, what would be the best way to prep for launching a new language here? Maybe updating the Crowdin download first to pull down the `translation.json` file for that language, then creating a PR like this?

**Comment by naomi-lgbt** [comment]
My proposal would be that when spinning up a new language, we would copy *all* the English JSON to the new directory in this PR.

Then, once this is in, we can update the workflow to enable Indonesian in a second PR.

After that, the translations will come in the next download.

File: config/i18n/locales/indonesian/serve.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "redirects": []
+}
```
Comment (naomi-lgbt) [inline config/i18n/locales/indonesian/serve.json]:
```suggestion
}

```
EOF NL should fix linter

Comment (scissorsneedfoodtoo) [reply config/i18n/locales/indonesian/serve.json]: Awesome, thanks for catching that @naomi-lgbt.

**Review by scissorsneedfoodtoo — APPROVED** [review]
Tested this by building Indonesian News locally and everything LGTM.

**Comment by ojeytonwilliams** [comment]
@scissorsneedfoodtoo this is safe to merge, right?  Am I correct in assuming that it only matter when we add this language to the deploy pipeline?

**Comment by raisedadead** [comment]
> ... Am I correct in assuming ...

See this for the guide that Kris is creating to address this: https://github.com/freeCodeCamp/freeCodeCamp/pull/48025/files

**Comment by scissorsneedfoodtoo** [comment]
@ojeytonwilliams, sorry, I forgot to remove the blocked label after the English `translation.json` file was added.

This should be safe to merge now. Test builds for Indonesian News work on my machine with everything that was added here and in https://github.com/freeCodeCamp/cdn/pull/163.

Here's a screenshot showing that localized dates are working:

image

And another showing CDN sourced footer:

image

**Review by scissorsneedfoodtoo — APPROVED** [review]
Approving here, but since I pushed a commit to fix some formatting, I'm not sure if I should merge it as well.

As far as I can tell, this is safe to merge, and we can add this to the deployment pipeline whenever we're ready.

**Comment by ojeytonwilliams** [comment]
> @ojeytonwilliams, sorry, I forgot to remove the blocked label after the English translation.json file was added.

No worries, @scissorsneedfoodtoo I just wasn't sure if this was waiting for something more than your review. I didn't want to merge it prematurely.

**Comment by scissorsneedfoodtoo** [comment]
Gonna go ahead and merge this now to clear the queue a bit.

Thanks again for all of your help with this @RafaelDavisH 👍

**Comment by scissorsneedfoodtoo** [comment]
This should be good to go now that the guide in the main repo has been merged, along with Rafael's other PR to the CDN repo 👍

**Review by raisedadead — APPROVED** [review]

**Pull request merged**